Emby.Server.Implementations/EntryPoints/UdpServerEntryPoint.cs

```diff
@@ -48,8 +48,16 @@ public sealed class UdpServerEntryPoint : IServerEntryPoint
         /// <inheritdoc />
         public Task RunAsync()
         {
-            _udpServer = new UdpServer(_logger, _appHost, _config);
-            _udpServer.Start(PortNumber, _cancellationTokenSource.Token);
+            try
+            {
+                _udpServer = new UdpServer(_logger, _appHost, _config);
+                _udpServer.Start(PortNumber, _cancellationTokenSource.Token);
+            }
+            catch (System.Net.Sockets.SocketException ex)
```

Comment:
```suggestion
            catch (SocketException ex)
```
And add `using System.Net.Sockets;` to the top